geth/account/utils.go

```diff
@@ -0,0 +1,33 @@
+package account
+
+import (
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// SelectedExtKey is a container for currently selected (logged in) account
+type SelectedExtKey struct {
```

Comment:
I would move code related to `SelectedExtKey` to a different file like `selected_ext_key.go`.